.github/workflows/check.yml

```diff
@@ -74,12 +74,17 @@ jobs:
         run: make typecheck-scripts
       - name: Start Postgres
         run: |
+          sudo apt-get install -y curl ca-certificates
+          sudo install -d /usr/share/postgresql-common/pgdg
+          sudo curl -o /usr/share/postgresql-common/pgdg/apt.postgresql.org.asc --fail https://www.postgresql.org/media/keys/ACCC4CF8.asc
+          sudo sh -c "echo 'deb [signed-by=/usr/share/postgresql-common/pgdg/apt.postgresql.org.asc] https://apt.postgresql.org/pub/repos/apt noble-pgdg main' > /etc/apt/sources.list.d/pgdg.list"
           sudo apt-get update
-          sudo apt-get install -y --no-install-recommends postgresql-16-pgvector
-          sudo sed -i 's/ local/ trust/g' /etc/postgresql/16/main/pg_hba.conf
-          sudo sed -i 's/ scram-sha-256/ trust/g' /etc/postgresql/16/main/pg_hba.conf
+          sudo apt-get purge -y man-db postgresql-16 postgresql-client-16
```

Comment:
The key thing to remove was the old server/client I think. Doing so should also clean up the old database, etc.